float32[] debris_position
float32[] debris_velocity
float32[] debris_relative_velocity
float32 debris_diameter
float32 time_of_detection
int32 object_count